content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -77,19 +80,26 @@ Example variables:
 **NOTE:  If you are using a GPU based AKS cluster (For example: AGENT_SIZE=Standard_NC6), you also need to [install the NVidia drivers](https://docs.microsoft.com/azure/aks/gpu-cluster#install-nvidia-drivers) on the cluster nodes before you can use GPUs with Kubeflow.**
```

Comment:
Line 80: Usually, on the Kubeflow site, the **Note:** paragraph doesn't have to be all bold.

```suggestion
**Note**:  If you are using a GPU-based AKS cluster (For example: AGENT_SIZE=Standard_NC6), you also need to [install the NVidia drivers](https://docs.microsoft.com/azure/aks/gpu-cluster#install-nvidia-drivers) on the cluster nodes before you can use GPUs with Kubeflow.
```

Reply:
fixed